apps/bfd-pipeline/bfd-pipeline-rda-bridge/pom.xml

```diff
@@ -21,6 +21,7 @@
         <checkstyle.skip>true</checkstyle.skip>
         <bfd-shared-utils.version>1.0.0-SNAPSHOT</bfd-shared-utils.version>
         <bfd-pipeline-rda-grpcs.version>1.0.0-SNAPSHOT</bfd-pipeline-rda-grpcs.version>
+        <checkstyle.skip>false</checkstyle.skip>
```

Comment:
Feel free to just remove this line now that my PR to remove the skip for the parent pom is in.